crates/server/src/services/transparency/log.rs

```diff
@@ -24,39 +26,50 @@ pub struct Summary {
     pub log_length: u32,
 }
 
-pub fn process(input: Input) -> Output {
-    let (summary_tx, summary_rx) = mpsc::channel::<Summary>(4);
-    let (log_data_tx, log_data_rx) = mpsc::channel::<LogLeaf>(4);
+pub fn spawn(input: Input) -> Output {
```

Comment:
I think the original distinction is that since this spawns a new task on (old 31/new 33) and doesn't do all the work itself we were calling it spawn. What sort of distinction are you following?

Reply:
Yeah, that's basically the reasoning behind the change; this function spawns a new task from the input, giving it the output of the spawned task (i.e. the task handle, senders, etc.).

The name `process` to me implies it's doing the work itself; i.e. something that should be called repeatedly to "do the thing".

Reply:
Oh, my bad. I read the diff backwards. LGTM